spec/fixtures/api_responses/courses.json

```diff
@@ -0,0 +1,1065 @@
+{
+  "data": [
+    {
+      "id": "12928386",
+      "type": "courses",
+      "attributes": {
+        "findable?": true,
+        "open_for_applications?": true,
+        "has_vacancies?": true,
+        "course_code": "2VMB",
+        "name": "Mathematics",
+        "study_mode": "full_time",
+        "qualification": "qts",
+        "description": "QTS full time with salary",
+        "content_status": "published",
+        "ucas_status": "running",
+        "funding_type": "salary",
+        "level": "secondary",
+        "is_send?": false,
+        "english": "must_have_qualification_at_application_time",
+        "maths": "must_have_qualification_at_application_time",
+        "science": null,
+        "gcse_subjects_required": [
+          "maths",
+          "english"
+        ],
+        "age_range_in_years": "11_to_18",
+        "accrediting_provider": {
+          "id": 13985,
+          "address4": "London",
+          "provider_name": "e-Qualitas",
+          "scheme_member": "is_a_UCAS_ITT_member",
+          "contact_name": "Programme Director",
+          "year_code": "2019",
+          "provider_code": "E69",
+          "provider_type": "scitt",
+          "postcode": "EC3N 3AJ",
+          "website": "http://www.e-qualitas.org.uk",
+          "address1": "6th Floor, 10 Lloyd's Avenue",
+          "address2": "",
+          "address3": "London",
+          "email": "initialteachertraining@e-qualitas.co.uk",
```

Comment:
we shouldn't using email address that not ours

i fact shouldn't this be dummy data... not want to see this pop up in a google search

Reply:
^^ If we're using fixtures they should still be dummy data

Reply:
Sorry.. my fault. I thought the data was anonymised.

Reply:
@defong @danielburnley 
QA, Staging and Prod data is not anonymised. It's public data - I’m not sure I see what the issue is?

Reply:
We only need dummy data for the structure.

Everything else is excessive.

Reply:
Data has been anonymised